Assignment-1-Calculator(Updated).c

```diff
@@ -0,0 +1,167 @@
+#include<stdio.h>
+
+#define ADD '+'
+#define SUBTRACT '-'
+#define MULTIPLY '*'
+#define DIVIDE '/'
+
+int getOperatorPriority(char operator);
+int validateInput(char input[]);
+void removeBlankSpaces(char input[]);
+void calculateOperations(int numbers[], char operators[], int *numberIndex, int *operatorIndex, int *errorStatus);
+int evaluateRemainingOperations(int numbers[], char operators[], int *numberIndex, int *operatorIndex, int *errorStatus);
+int performCalculation(char input[], int *errorStatus);
+
+/* Main Function */
+int main() {

```

Comment:
no need of separate new file, you should edit the same file for fixing review comments and need to push the code, please refer any online source like how to fix the review comments and push the code.